Assets/Mirror/Examples/MultipleMatches/Scripts/MatchController.cs

```diff
@@ -0,0 +1,314 @@
+﻿using System;
+using System.Collections;
+using System.Collections.Generic;
+using UnityEngine;
+using UnityEngine.UI;
+
+namespace Mirror.Examples.MultipleMatch
+{
+    [RequireComponent(typeof(NetworkMatchChecker))]
+    public class MatchController : NetworkBehaviour
+    {
+        internal readonly SyncDictionary<NetworkIdentity, MatchPlayerData> matchPlayerData = new SyncDictionary<NetworkIdentity, MatchPlayerData>();
+        internal readonly Dictionary<CellValue, CellGUI> MatchCells = new Dictionary<CellValue, CellGUI>();
+
+        CellValue boardScore = CellValue.None;
+        bool playAgain = false;
+
+        [Header("GUI References")]
+        [SerializeField] internal CanvasGroup canvasGroup;
+        [SerializeField] Text gameText;
+        [SerializeField] Button exitButton;
+        [SerializeField] Button playAgainButton;
+        [SerializeField] Text winCountLocal;
+        [SerializeField] Text winCountOpponent;
+
+        [Header("Diagnostics - Do Not Modify")]
+        [SerializeField] internal CanvasController canvasController;
```

Comment:
this is not what internal is for.
please just make the fields public instead of SerializeField internal.
keep it simple :)